eng/pipelines/runtime.yml

```diff
@@ -876,7 +876,8 @@ jobs:
     runtimeFlavor: mono
     buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
     platforms:
-    - Windows_NT_x64
+    # Reenable when https://github.com/dotnet/runtime/issues/34582 is fixed.
```

Comment:
This should be updated to use `/p:TestRunner=xunit` and re-enabled.

Reply:
I only added the infra to enable the runner locally. I validated that the xunit runner works locally and I documented how to invoke it. It's integrated into the existing infrastructure (what you asked for) instead of invoking it via script or manually to minimize steps to make it run. To cite @danmosemsft from one of his responses:

> Running xunit.console manually – this would no longer be CI protected, however it should work fine as we’re not changing the runner/test assembly interface – vstest relies on the same interface.

Aside from that, @stephentoub spoke with @marek-safar offline and they believe it's fine to disable the Windows leg for Mono until the thread pool issue is fixed.

Reply:
I think it is important to protect that this switch works in the CI.

Reply:
> I think it is important to protect that this switch works in the CI.

Why do you think so?

Reply:
I believe that it will be permanently broken otherwise.

Reply:
I added CI protection for the `/p:TestRunner=xunit` switch.